admin_manual/configuration/files/encryption_configuration_quick_guide.rst

```diff
@@ -0,0 +1,77 @@
+====================================
+Encryption Configuration Quick Guide
+====================================
+ 
+Encryption Types
+----------------
+
+ownCloud provides two encryption types:
+
+- **User specific Key:** every user has their own private/public key pairs, and the private key is protected by the user's password.
+
+- **Master Key:** there is only one key (or key pair) and all files are encrypted using that key pair.
+  
+How To Enable Encryption
+------------------------
+
+It's your choice what type of encryption you want to use.
```

Comment:
This sentence is not necessary, as it leads to a wider discussion who allows/decides what.
Maybe better to add: It has to be decided which kind of encryption will be taken as only one at a time can be used

Reply:
Well said.